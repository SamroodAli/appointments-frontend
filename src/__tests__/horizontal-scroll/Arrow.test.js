import renderer from 'react-test-renderer';
// import { render } from '@testing-library/react';
import Arrow from '../../components/horizontal-scroll/Arrow';
import Wrapper from '../../testWrapper';

test('Arrow snapshot testing', () => {
  const tree = renderer.create(
    <Wrapper>
      <Arrow className="arrow">
        Left Arrow
      </Arrow>
    </Wrapper>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
