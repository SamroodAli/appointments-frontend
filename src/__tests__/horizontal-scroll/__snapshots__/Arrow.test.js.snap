// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Arrow snapshot testing 1`] = `
<div
  className="arrow-container arrow-container"
>
  <button
    className="arrow arrow"
    disabled={false}
    onClick={null}
    type="button"
  >
    Left Arrow
  </button>
</div>
`;
